db/system_keyspace.cc

```diff
@@ -2231,6 +2250,25 @@ std::vector<schema_ptr> system_keyspace::auth_tables() {
     return {roles(), role_members(), role_attributes(), role_permissions()};
 }
 
+std::vector<schema_ptr> system_keyspace::group0_tables() {
+    std::vector<schema_ptr> r;
+    auto auth_tables = system_keyspace::auth_tables();
+    std::copy(auth_tables.begin(), auth_tables.end(), std::back_inserter(r));
+    r.insert(r.end(),
```

Comment:
is `repair_history` really a group 0 table?

`raft`, `raft_snapshots`, `raft_snapshot_config` definitely are not

the list is also missing many important tables, like everything in `system_schema`